tests/nfv/prepare_env.pm

```diff
@@ -33,8 +33,8 @@ sub run {
     zypper_call('in git-core openvswitch-switch dpdk qemu tcpdump', timeout => 200);
 
     # Clone repositories
-    assert_script_run("git clone $vsperf_repo");
-    assert_script_run("git clone $dpdk_repo");
+    assert_script_run("git clone --depth 1 $vsperf_repo");
```

Comment:
you using git clone a lot , also it is used by other guys . what about introducing utils function which will wrap around this logic ?

Reply:
That´s a good idea indeed. I can work on it, but maybe for a different patch :)